### DISCUSSION TITLE
知乎主网站及专栏渲染不一致问题

### DISCUSSION THREAD

**Discussion by HelpFireCode** [F9y4ng/GreasyFork-Scripts#242] [问答专区(Question / Answer)] [answered]
我使用的字体是思源黑体，Source Han Sans SC 。
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/75774948/6cd36151-5604-4730-b664-577d0c2fdf74)
如图是知乎主网站字体，没有渲染。
而搜索引擎打开的知乎专栏界面渲染成功。
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/75774948/c1df269a-a051-4c44-99a3-c5de703dae05)
这个是什么问题呢？

**Comment by F9y4ng** [comment, marked as answer]
1. 首先需要排除的就是与其他第三方脚本的冲突。（在仅开启字体渲染脚本（如有知乎相关扩展也关闭）的情况下，清空缓存后重试）
2. 排除脚本加载不及时或不成功的问题。（[可尝试以下办法](https://github.com/F9y4ng/GreasyFork-Scripts#%E5%85%B3%E4%BA%8E%E8%84%9A%E6%9C%AC%E5%8A%A0%E8%BD%BD%E5%BB%B6%E8%BF%9F%E7%9A%84%E5%A4%84%E7%90%86%E5%8A%9E%E6%B3%95)）
3. 你遇到的问题，我本地不能复现。

**Reply by HelpFireCode** [reply]
感谢回复！
1.没有第三方脚本问题，因为以上情况我只用了字体渲染一个脚本。
2.每次更换字体时，采用①刷新②将浏览器退出以更新。
3.我已解决，过程是修改自定义字体为：{"ch":"Noto Sans S Chinese","en":"Noto Sans S Chinese"}，接着选择，即可。
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/75774948/155c369e-c10e-4bf8-8de7-8a3e7c9df8e1)

**Reply by F9y4ng** [reply]
添加新字体，最好是到`www.fontke.com`搜索找到你想要的字体，然后下载字体，再按照 #64 中的方法获取自定义字体名称。

因为即使是同一字体家族的字体也会有字体名称不同的情况，一切以你下载的字体所定义的字体名称为准。（其他第三方的字体下载有些会修改字体参数改变字体名称或字体属性，造成脚本或浏览器无法识别或正确解析该字体。）
